firefox_privacy_notice/pl.md

```diff
@@ -53,7 +53,7 @@ Firefox automatycznie nawiązuje połączenia z nami i naszymi dostawcami usł
 
 * Miniatury
 
-	Miniatury to funkcja przeglądarki Firefox wyświetlana na stronach nowych kart. Aby udostępniać funkcję miniatur, Firefox przesyła do firmy Mozilla powiązane z nimi dane, takie jak liczba kliknięć i wyświetleń, adres IP użytkownika, informacje o ustawieniach regionalnych i dane miniatur (np. położenie i rozmiar siatki). W przeglądarce Firefox Beta pewne krótkoterminowe eksperymenty związane z Telemetrią (patrz wyżej) dla funkcji Miniatury mogą zbierać informacje o często odwiedzanych domenach.
+	Podpowiedzi wyszukiwania to funkcja, która pomaga znaleźć hasła, których często szukali inni użytkownicy. Podpowiedzi są dostarczane przez domyślną wyszukiwarkę (np. Google, Yahoo itp.), a nie przez Firefoksa. Jeśli używana domyślna wyszukiwarka obsługuje podpowiedzi, po włączeniu tej funkcji Firefox może wysyłać hasła wpisywane w pasku Awesome Bar lub w pasku wyszukiwania do domyślnej wyszukiwarki, by pobrać podpowiedzi. Podlega to Polityce prywatności domyślnej wyszukiwarki. [Więcej informacji o podpowiedziach wyszukiwania można znaleźć tutaj](https://support.mozilla.org/kb/use-popular-search-suggestions-firefox-search-bar), razem z informacjami, jak włączyć i wyłączyć tę funkcję.
```

Comment:
Awesome Bar is the marketing term of the location bar. IMO this document should avoid using it.